sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -310,6 +315,14 @@ def send(self, message):
                 :caption: Send message.
 
         """
+        try:
+            batch = self.create_batch()
+            for each in message:
+                batch.add(each)
```

Comment:
I'm leaning towards making it a private method on BatchMessage object to make the code reusable and maintainable.
like:
```
batch_message = self.create_batch()
batch_message._from_list(messages)
```

Reply:
It's funny you say that, initially I wrote this as create_batch taking a list arg, but didn't want to break us into jail with the rest of the crew when they found out we grew another feature.  I like your approach.

Reply:
I also like this way. I'm doing the same in EH with Adam's suggestion.
But how about using _load_list, _load_from_list or other method name but not _from_list? We use "from_xxx" for some class methods. This is an instance method.